README.md

```diff
@@ -35,7 +35,7 @@ This second step needs to be repeated each time you start a new shell.
 
 2.  Generate the textproto library:
 
-        make -C latin_scansion/lib
+        make -C latin_scansion
```

Comment:
is this changing the location of textproto files?

Reply:
No, that happened in some earlier PR, the documentation is just out of date, as I noticed when I went to test this.